.github/ISSUE_TEMPLATE/release-check-list.md

```diff
@@ -70,29 +72,36 @@ Identify/create the bundles that we will release for Kata and enclave-cc.
     If you absolutely cannot use a Kata release,
     you can consider releasing one of these bundles.
 
+- [ ] 3. :eyes: **Create a peer pods release**
+
+    Update peer-pods to use the release version of kata-containers and the latest attestation-agent, and test it, following the [release candidate testing process](https://github.com/confidential-containers/cloud-api-adaptor/blob/main/docs/Release-Process.md#release-candidate-testing).
+
+    Cut a release following the [documented flow](https://github.com/confidential-containers/cloud-api-adaptor/blob/main/docs/Release-Process.md#cutting-releases).
```

Comment:
I think we could simplify this by saying something like:

> Create a peer pods release release based on the Kata release, by following the [documented flow](https://github.com/confidential-containers/cloud-api-adaptor/blob/main/docs/Release-Process.md).

The release candidate and then release stage is covered in that guide, so might not need mentioning here?